google/cloud/aiplatform/metadata/artifact.py

```diff
@@ -327,60 +326,15 @@ def create(
             credentials=credentials,
         )
 
-    @classmethod
```

Comment:
Should the title be `feat!` to do a major bump rather than only mentioning that it's a breaking change?

Reply:
done. PR title is set to Feat: ...

Reply:
`feat` will do a minor version bump (1.15.1 -> 1.16.0), `feat!` with the exclamation mark will do a major version bump (1.15.1 -> 2.0.0).

Reply:
Since it's already been merged it won't have any impact on `release-please` bot. Could you let the client library maintainers for this repo know about this, and see if they'd like to do a minor version bump vs major version bump?